scio-core/src/main/scala/com/spotify/scio/coders/instances/JavaCoders.scala

```diff
@@ -112,6 +115,10 @@ trait JavaCoders extends JavaBeanCoders {
 
   implicit def coderJEnum[E <: java.lang.Enum[E]: ClassTag]: Coder[E] =
     Coder.beam(SerializableCoder.of(ScioUtil.classOf[E]))
+
+  implicit val readableFileCoder: Coder[ReadableFile] = Coder.beam(new ReadableFileCoder())
```

Comment:
@davidvspot Actually these should be moved to `BeamTypeCoders` and it will fix your issue with these coders not being found. Sorry about the back and forth.